server/src/com/desertkun/brainout/playstate/ServerPSGame.java

```diff
@@ -197,12 +199,16 @@ private void inited()
             if (client instanceof PlayerClient)
             {
                 PlayerClient playerClient = ((PlayerClient) client);
-
-                if (playerClient.isInitialized())
-                {
-                    playerClient.setModePayload(
-                        ((ServerRealization) getMode().getRealization()).newPlayerPayload(playerClient));
-                }
+                // Wait 1 second to let player's state to change
+                BrainOut.Timer.schedule(new TimerTask() {
```

Comment:
Can we schedule the whole `for (ObjectMap.Entry<Integer, Client> entry : C.getClients())` loop once instead of scheduling one for every player?